src/language.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+class _Language(object):
+  def __init__(self, local_name, lang_code, region_code, emoji_flag):
+    self._local_name = local_name
+    self._lang_code = lang_code
+    self._region_code = region_code
```

Comment:
Why do we handle region code ? For countries flags ? 
If it is, language switcher best practices don't recommend displaying countries flags when targeting generic languages.
Example : English speaking people outside US will be confused to see the US flag in the language selector.

Reply:
Because the content is written in American-style English. For example we would spell "color" rather than "colour".

Reply:
In any case, I'm happy to follow up with UX improvements to the language switcher if we're not adhering to best practices. Feel free to file a new issue. cc @tanhengyeow

Reply:
The lang in hreflang (or in html lang) is about to say which population am i targeting with this page : 
- Language speaking (en, fr, ja, es...etc) ? 
- Language speaking who are based somewhere (en-us, en-uk, fr-ca).
Search engines use these informations (Google uses hreflang, Bing uses html lang) to provide the page to users based on ths targeting. In our case i believe we should keep on generic language targeting.

Reply:
Gotcha, thanks for explaining. Could you file a new issue so we can discuss those changes? For now, it shouldn't affect this PR, which is mostly just code organization.